components/nimbus/build.rs

```diff
@@ -8,6 +8,9 @@ pub fn main() {
     #[cfg(feature = "stateful-uniffi-bindings")]
     uniffi::generate_scaffolding("./src/nimbus.udl").unwrap();
 
+    #[cfg(not(feature = "stateful-uniffi-bindings"))]
```

Comment:
This is... eh. I'm not super pleased with this but unless we want to pull Cirrus out into its own crate we can't really make this any better as far as I can tell.